tdex/docker-compose.yml

```diff
@@ -6,31 +6,52 @@ services:
       APP_HOST: tdex_caddy_1
       APP_PORT: 8080
 
+  oceand:
+    image: ghcr.io/vulpemventures/oceand:v0.1.17@sha256:9a7e1b8df4d7907ad66e3475c5a5f357d73c49df96214c00eba121f2aea9a50d
```

Comment:
I'm sorry, where did you get this digest from? I can't find it [here](https://github.com/vulpemventures/ocean/pkgs/container/oceand)

Reply:
When you click on the `...` next to `Digest` it shows you the multi-arch digest:

<img width="741" alt="image" src="https://github.com/getumbrel/umbrel-apps/assets/85373263/f3f11272-f439-49b7-8722-7ca2a1f080da">

This works because the v0.1.17 image is multi-arch as seen here (both amd64 and arm64):

<img width="766" alt="image" src="https://github.com/getumbrel/umbrel-apps/assets/85373263/afb4115e-47ed-46dd-82ff-9ac940174498">

Another way you can get the correct digest for multi-arch images is to run `docker pull <image><tag>` and the correct digest will be shown in the output. For example:

```
$ sudo docker pull ghcr.io/vulpemventures/oceand:v0.1.17
v0.1.17: Pulling from vulpemventures/oceand
...
Digest: sha256:37b813b43b068e32d918b9f42539c33287308cc81d00909c742b7164dfefc0c4
Status: Downloaded newer image for ghcr.io/vulpemventures/oceand:v0.1.17
ghcr.io/vulpemventures/oceand:v0.1.17
```

Reply:
done.